thrive-her/thrive-her-mvp
# build and style dashboard page

| Type
-- | --
✓ | ✨ New feature

**Description**

- Created and styled the dashboard page (homepage) to route to events, forum, Crisis Text Line, and Events pages.
- Created a reusable logout button
- Updated the reusable banner with branding and links to different pages
- Updated styling of the page before you log in (You must be logged in and prompt the user to log in throughout different pages)
- Added role and aria-label to make it more accessible
- Utilized usePassageUserInfo to access first name to make the user experience more personal
- Investigated and added first and last name to register page 


https://github.com/thrive-her/thrive-her-mvp/assets/112290188/16be45f4-7056-4ef1-8221-8c0766531602


https://github.com/thrive-her/thrive-her-mvp/assets/112290188/20e25e7f-3802-4982-b300-c5a356158c51

<img width="444" alt="Sign up form" src="https://github.com/thrive-her/thrive-her-mvp/assets/112290188/79b507bd-ab0f-4fc8-9397-cabf130ed830">



closes #6 

TODOs:
**ALL: Before you test: Please register with a new email (+ type in your first name and your last name) or remove yourself from your console and register again (+ type in your first name and your last name). Otherwise, we will run into errors because metadata doesn't exist.** 

Testing Steps / QA Criteria
From your terminal, pull down this branch then npm install to install the newly added dependencies locally, and start to launch the app.

- Log in and then check out dashboard page and click on if the routings are working properly 
- Log out and try to go to different endpoints to see if "Please log in" message and login button are consistent
